src/lib/components/consignments/components/boolean_button.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from "react"
```

Comment:
It seems like the way a consumer might want to use this component, is to basically provide the left/right text. So will this component be handling the 'toggle'?

ie- should this component do the UI toggling, and then call the `onPress` the client passed in (since presumably that might trigger something)?